packages/sdk/chainDeploy/helpers/oracles/redstone.ts

```diff
@@ -0,0 +1,33 @@
+import { RedstoneAdapterPriceOracle } from "../../../typechain/RedstoneAdapterPriceOracle";
+import { RedStoneDeployFnParams } from "../types";
+
+import { addUnderlyingsToMpo } from "./utils";
+
+export const deployRedStonePriceOracle = async ({
+  ethers,
+  getNamedAccounts,
+  deployments,
+  redStoneAddress,
+  redStoneAssets
+}: RedStoneDeployFnParams): Promise<{ redStoneOracle: RedstoneAdapterPriceOracle }> => {
+  const { deployer } = await getNamedAccounts();
+
+  const mpo = await ethers.getContract("MasterPriceOracle", deployer);
+
+  //// RedStone Oracle
+  const redStone = await deployments.deploy("RedStonePriceOracle", {
+    from: deployer,
+    args: [redStoneAddress],
+    log: true,
+    waitConfirmations: 1
+  });
+
+  if (redStone.transactionHash) await ethers.provider.waitForTransaction(redStone.transactionHash);
+  console.log("RedStonePriceOracle: ", redStone.address);
+
+  const redStoneOracle = (await ethers.getContract("RedStonePriceOracle", deployer)) as RedstoneAdapterPriceOracle;
```

Comment:
let's keep the convention - the contract name is the deployment artifact name?
RedStonePriceOracle should be RedstoneAdapterPriceOracle

Reply:
Thx I don't think it would have worked otherwise.